nucypher/characters/control/controllers.py

```diff
@@ -248,6 +249,7 @@ def test_client(self):
     def make_control_transport(self):
 
         self._transport = Flask(self.app_name)
+        self._transport.config['MAX_CONTENT_LENGTH'] = MAX_UPLOAD_CONTENT_LENGTH
```

Comment:
Why is the constant passed in both of these places (`make_control_transport` and `make_rest_app`)?

Reply:
Those are just different names for the same things in different places AFAICT